route4me-csharp-sdk/Route4MeSdkV5UnitTest/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "settings": {
-    "actualApiKey": "11111111111111111111111111111111",
+    "actualApiKey": "5B05A8B6C4DB22C0CD6966F01D5FB4E0",
```

Comment:
actual ApikKey's value should not be available in git

Reply:
Yes, I could not notice it as many files were changed at once.

Reply:
done